MessageGeneration/README.md

```diff
@@ -0,0 +1,5 @@
+# MessageGeneration
+
+[.NET](https://www.microsoft.com/net) Library for generating C# source code for [ROS](http://www.ros.org/) message/service/action.
+
+Forked from: [ROS#](https://github.com/siemens/ros-sharp)
```

Comment:
Missing licensing here? (Not sure, haven't forked other repos into my own before)

Reply:
Wouldn't worry that much, I think this is sufficient